readme.md

```diff
@@ -291,6 +292,25 @@ this disorder.
 
 [R: medicinet.com](https://www.medicinenet.com/munchausen_syndrome_by_proxy/article.htm)
 
+
+#### [Lima Syndrome](https://en.wikipedia.org/wiki/Stockholm_syndrome#Lima_syndrome)
```

Comment:
Since this is closely related to Stockholm Syndrome, please add this entry immediately after that entry.